tests/dummy/app/services/keyboard.js

```diff
@@ -0,0 +1,86 @@
+import { action } from '@ember/object';
```

Comment:
this allows us to keep documentation and interactive widgets/examples for 6.x

Basically deprecated/removed functionality for old propagation model/priority based responders moved from add-on code to dummy app for the purpose of interactive docs only.